react/modules/sample/src/pages/employee/SampleSearchWithMultipleTabs.js

```
@@ -0,0 +1,63 @@
+import React, { useState, useEffect } from "react";
+import { useTranslation } from "react-i18next";
+import { InboxSearchComposer, Loader } from "@egovernments/digit-ui-components";
+import { InboxConfig } from "../../configs/SampleSearchWithMultipleTabsConfig";
+
+const SampleSearchWithMultipleTabs = ({}) => {
```

Biome (1.9.4)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)